### DISCUSSION TITLE
Is it possible to specify position / move the chat widget?

### DISCUSSION THREAD

**Discussion by jwld** [chatwoot/chatwoot#2145] [General]
We're using the SDK and it's always in the bottom right which blocks some important parts of our app.

**Comment by sojan-official** [comment]
you could place the button either left or right using the position attribute

```
window.chatwootSettings = {
  hideMessageBubble: false,
  position: 'left', // This can be left or right
  locale: 'en', // Language to be set
  type: 'standard', // [standard, expanded_bubble]
};
```

you can also trigger the widget from a custom element using : https://www.chatwoot.com/docs/website-sdk#to-trigger-widget-without-displaying-bubble

**Reply by jwld** [reply]
Thanks, that sort of works. To go into more detail - it's better for us to have it on the right, but all our modals are on the right and it blocks the buttons when they're open. Ideally we could move it to the left when a modal is opened, but it seems the chatwoot settings are only applied on load?

**Reply by pranavrajs** [reply]
@jwld Yes, that is correct. You cannot apply the settings after the widget is loaded.
